.gitattributes

```diff
@@ -1,3 +1,7 @@
 
 * annex.backend=MD5E
-**/.git* annex.largefiles=nothing
\ No newline at end of file
+**/.git* annex.largefiles=nothing
+images/brain-extraction annex.largefiles=nothing
+images/brain-generation annex.largefiles=nothing
+* annex.largefiles=((mimeencoding=binary)and(largerthan=0))
+** annex.largefiles=(not (include=*.py))
```

Comment:
perhaps add yaml/json/text/markdown files to this as well.

Reply:
and Dockerfiles

Reply:
I think since they are considered as text, they are excluded by `* annex.largefiles=((mimeencoding=binary)and(largerthan=0))`. they are not annexed now. but I can add them just to be in the safe side.